vrp-pragmatic/src/format/solution/geo_serializer.rs

```diff
@@ -32,6 +32,12 @@ struct FeatureCollection {
     pub features: Vec<Feature>,
 }
 
+#[derive(Clone, Debug, Serialize, PartialEq)]
+struct FeatureCollectionWithSolution {
+    geo: FeatureCollection,
+    pragmatic: Solution,
```

Comment:
currently, I'm revisiting this approach: there is already `extras` property in solution which supposed to keep extra information. I see two ways to use it:
- either add a direct optional property to keep FeatureCollection. But this will make geojson types everywhere. Not sure that I want that
- or tweak output directly to inject it in extras using custom serializer as example